hover/src/main/java/io/mattcarroll/hover/defaulthovermenu/view/ViewHoverMenu.java

```diff
@@ -50,6 +50,25 @@
     private HoverMenuAdapter mAdapter;
     private SharedPreferences mPrefs;
     private Set<OnExitListener> mOnExitListeners = new HashSet<>();
+    private Set<OnCollapseAndExpandListener> mOnCollapseAndExpandListeners = new HashSet<>();
```

Comment:
Can you change the implementation to a CopyOnWriteArraySet to avoid possible concurrency issues?